samples/drivers/crypto/sample.yaml

```diff
@@ -21,6 +21,7 @@ tests:
         - ".*: GCM Mode"
   sample.drivers.crypto.mbedtls.micro:
     tags: crypto
+    platform_exclude: qemu_cortex_a53
```

Comment:
Why is this necessary? Is there a technical reason why this test should not/cannot run on the `qemu_cortex_a53`?

Reply:
nvm, build failure because I forgot `west update`. Pushing v2 without this line.